packages/react/src/components/ComboBox/ComboBox.stories.js

```diff
@@ -106,9 +103,126 @@ export const WithLayer = (args) => (
           placeholder="Filter..."
           titleText="Third Layer"
           helperText="Combobox helper text"
-          {...args}
         />
       </Layer>
     </Layer>
   </div>
 );
+
+export const Playground = (args) => (
+  <div style={{ width: 300 }}>
+    <ComboBox
+      onChange={() => {}}
+      id="carbon-combobox"
+      items={items}
+      downshiftProps={{
+        onStateChange: () => {
+          console.log('the state has changed');
+        },
+      }}
+      itemToString={(item) => (item ? item.text : '')}
+      placeholder="Filter..."
+      titleText="ComboBox title"
+      helperText="Combobox helper text"
+      {...args}
+    />
+  </div>
+);
+
+Playground.argTypes = {
+  ariaLabel: {
+    table: {
+      disable: true,
+    },
+  },
+  id: {
+    table: {
+      disable: true,
+    },
+  },
+  downshiftProps: {
+    table: {
+      disable: true,
+    },
+  },
+  initialSelectedItem: {
+    table: {
+      disable: true,
+    },
+  },
+  invalidText: {
+    table: {
+      disable: true,
+    },
+  },
+  items: {
+    table: {
+      disable: true,
+    },
+  },
+  itemToElement: {
+    table: {
+      disable: true,
+    },
+  },
+  itemToString: {
+    table: {
+      disable: true,
+    },
+  },
+  light: {
+    table: {
+      disable: true,
+    },
+  },
+  onChange: {
+    table: {
+      disable: true,
+    },
+  },
+  onInputChange: {
+    table: {
+      disable: true,
+    },
+  },
+  onStateChange: {
+    table: {
+      disable: true,
+    },
+  },
+  onToggleClick: {
+    table: {
+      disable: true,
+    },
+  },
+  selectedItem: {
+    table: {
+      disable: true,
+    },
+  },
+  shouldFilterItem: {
+    table: {
+      disable: true,
+    },
+  },
+  translateWithId: {
+    table: {
+      disable: true,
+    },
+  },
+  titleText: {
+    table: {
+      disable: true,
+    },
+  },
+  type: {
+    table: {
+      disable: true,
```

Comment:
What's the reason for excluding this one?

Reply:
I've updated the others this one specifically was that I couldnt get it to work even using a select control on it with both variants

Reply:
Sounds good 👍